CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ Create a new [free developer instance](https://developer.atlassian.com/platform/
 Create a new [GitHub App](https://github.com/settings/apps), setting the following config:
 
 - **GitHub App name**: Anything you want, but it must be unique across GitHub.
-- **Homepage URL**: `https://DOMAIN`
+- **Homepage URL**: `https://github.com/apps/**GitHub App name**`
```

Comment:
Updating **DOMAIN** is a great call! always seems to be the ambiguous part 🤔 

The homepage URL should be the local domain
e.g. https://joshs-sweet-tunnel.public.atlastunnel.com
or for the Ngrok among us https://8a74-101-100-131-65.ngrok.io

Reply:
Checked with @rachellerathbone, **HOMEPAGE URL** is not the tunnelled URL, its the URL of the Github Website, i.e. `https://github.com/apps/APPNAME`